athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/connection/EnvironmentProperties.java

```diff
@@ -95,7 +95,7 @@ private Map<String, String> authenticationConfigurationToMap(AuthenticationConfi
     {
         Map<String, String> authMap = new HashMap<>();
 
-        if (StringUtils.isNotBlank(auth.secretArn())) {
+        if (auth != null && StringUtils.isNotBlank(auth.secretArn())) {
```

Comment:
With the previous change, auth being null throws an exception. Creating a new code path here, what happens if authMap is empty for calling methods?

Reply:
no tests covering. empty was expected before but it was just reaching this null exception before. empty is handled properly on line 57